sklearn/tests/test_base.py

```diff
@@ -436,6 +436,7 @@ def __init__(self, attribute_pickled=5):
 
     def __getstate__(self):
         data = self.__dict__.copy()
+        data["_sklearn_version"] = sklearn.__version__
```

Comment:
I am not entirely sure this is the right fix. I don't remember whether this test was supposed to test estimators outside scikit-learn or not. @HolgerPeters do you remember off the top of your head?

A closer look at #8324 may be needed.